### DISCUSSION TITLE
GitHub email verification codes not arriving

### DISCUSSION THREAD

**Discussion by purvidas** [community/community#118995] [New to GitHub] [answered]
I attempted to log into my account but found that I couldn't receive the verification codes over mail, even after hours. I've searched everywhere including checking spam folders, with no luck. Upon researching if others were facing similar issues, I discovered that a majority of users are unable to sign in due to this problem for over a day now. Some users receive the email after hours, and by the time they receive it, the code has already expired expired, so the loop continues. Even the forgot password option doesn't resolve the issue. While I was able to finally access using a passkey but the verification mails still doesn't work. Could you please investigate?

**Comment by Metal-joker** [comment]
same here, I use proton mail

**Reply by bkutasi** [reply]
Same here, i use gmail and cloudflare forwarding. Never received anything.

**Reply by rudden** [reply]
Same, I use zohomail. I receive the emails several hours after I've requested them. Been a problem since yesterday, extremely irritating since I can't login to my other account.

**Reply by CODE-Y02** [reply]
same here, using outlook I received few one but after hours, codes was already expired at that time.

**Reply by santropedro** [reply]
https://www.reddit.com/r/github/comments/1c3386x/github_email_verification_codes_not_arriving/
https://www.reddit.com/r/github/comments/1c3efdt/email_verification_codes_not_arriving/
https://www.reddit.com/r/github/comments/1c2fuse/verification_codes_dont_work/
These people can't comment or upvote here or create issues because they can't log in. Can they now, has this been resolved?

**Reply by EatsShootsAndLGBTs** [reply, marked as answer]
Same here, using outlook. Been like that for 2 days. Received perhaps three dozen verification codes that had all expired by the time they arrived, until a few moments ago when one arrived moments after it was send, and was accepted. So perhaps good news for others who have been experiencing similar problems.

**Comment by mir6ux** [comment]
same here, the codes come after hours and expired and i cant login so i had to create this new account

**Comment by DevRehannaLP** [comment]
I'd thought I would end up being the only one with a bad few past days.  Keep your head up everyone we got this!

**Comment by DevRehannaLP** [comment]
I'd thought I would end up being the only one with a bad few past days.  Keep your head up everyone we got this!

**Comment by Li-Major** [comment]
same, icloud, gmail, outlook, none works

**Comment by MsCalico59-Cheetah77** [comment]
Same here none works

**Comment by christophervistal25** [comment]
Same here I can't receive email verification code!

**Comment by tiararodney** [comment]
Hi,

After jumping to conclusions too quickly, the GitHub support kindly made me aware that the fault was on my end, where the e-mail address simply did not exist. I was expecting a *Catch-All* configuration to exist, which didn't.

In case, this might be helpful to someone else.
